package.json

```diff
@@ -91,6 +91,15 @@
     "url": "https://github.com/ExchangeUnion/xud/issues"
   },
   "homepage": "https://github.com/ExchangeUnion/xud#readme",
+  "files": [
```

Comment:
Is this a whitelist for files to be uploaded to NPM or should we add an additional `.npmignore` to make sure that the `lib` folder doesn't get distributed?

Reply:
It's a whitelist, I tried to use an npmignore file but it was not working as expected. While googling to try to fix it I came across this which convinced me to go the whitelist approach instead. https://medium.com/@jdxcode/for-the-love-of-god-dont-use-npmignore-f93c08909d8d